lib/doorkeeper/oauth/pre_authorization.rb

```diff
@@ -15,7 +15,10 @@ class PreAuthorization
                     :code_challenge, :code_challenge_method
       attr_writer   :scope
 
-      def initialize(server, client, attrs = {})
+      def initialize(server, client, params)
```

Comment:
Actually, I think it must be done on a controller lever, not in the library classes itself.

Reply:
@nbulaj  I feel so, just tried to keep the knowledge about the list of params in single place.

It's not critical at all for me to move `.permit` call to controller, please confirm that I should )

Reply:
For sure, it would be great, thanks!

Reply:
Done! )